### DISCUSSION TITLE
Adding Playerbot

### DISCUSSION THREAD

**Discussion by bleedingfurball** [azerothcore/azerothcore-wotlk#18426] [Modules]
I have been trying to wrap my head around how to add playerbots. 

"Please kindly note that this requires some custom changes to AzerothCore, so it is necessary to compile with a custom branch from my fork: https://github.com/ZhengPeiRu21/azerothcore-wotlk/tree/Playerbot"

I presume I add in the actual playerbot module into my azerothcore/modules folder. I'm not 100% sure what to do with trying to add a custom branch with his fork. 

![playerbot](https://github.com/azerothcore/azerothcore-wotlk/assets/161189766/25b8a734-d771-42f3-90e1-291866198cf6)

It looks different from when I first started, been messing with stuff, and don't know how to revert everything. 

Is there a step by step guide to doing this? I feel dumb, I can usually figure stuff out but for some reason this is stumping me. 

If someone could guide me on exactly what to press and such, that would be awesome. I greatly appreciate all help, and would be forever grateful.

**Comment by sudlud** [comment]
Instead of checking out the "clean" / "main" AC git repository, you must checkout the AC git repository from the link you already found, which is a fork of AC adapted to work with the playerbots.

(A fork is basically a copy of the main repository which allows custom changes.)

**Reply by abutbul** [reply]
use trickerer's AC bots (under the TC bots repo) he has a patch file that works on the latest HEAD from AC as well.
https://github.com/trickerer/Trinity-Bots

**Comment by xengine-qyt** [comment]
there is playerbot :https://github.com/liyunfan1223/mod-playerbots
